packages/suite/src/config/wallet/menu.tsx

```diff
@@ -35,9 +35,11 @@ export const ITEMS = [
         title: <Translation id="TR_NAV_SIGN_AND_VERIFY" />,
         icon: 'SIGN',
         isHidden: (_account: Account) => {
-            return true;
-            // const network = NETWORKS.find(n => n.symbol === account.symbol);
-            // return !(network && network.testnet && network.hasSignVerify);
+            const network = NETWORKS.find(n => n.symbol === _account.symbol);
+            return !(
+                (network?.networkType === 'bitcoin' || network?.networkType === 'ethereum') &&
```

Comment:
`return !(network && network.testnet && network.hasSignVerify)` vas valid 
`network?.networkType === 'bitcoin' || network?.networkType === 'ethereum'` is unnecessary